k8s/core/configmap/configmap_lock_v1.go

```diff
@@ -153,7 +156,6 @@ func (c *configMap) refreshLockV1(id string) {
 	for {
 		select {
 		case <-refresh.C:
-			c.kLockV1.Lock()
```

Comment:
Keep the lock here and remove the lock in tryLockV1.
c.kLockV1.unlocked is to be read under a lock.

This lock re-arrangement is not required anymore since we are disallowing Unlock from FatalCB

Reply:
done